log4j-api/src/main/java/org/apache/logging/log4j/Logger.java

```diff
@@ -292,12 +294,12 @@
 
     /**
      * Logs a message at the {@link Level#DEBUG DEBUG} level including the stack trace of the {@link Throwable}
-     * <code>t</code> passed as parameter.
+     * <code>throwable</code> passed as parameter.
      *
      * @param message the message to log.
-     * @param t the exception to log, including its stack trace.
+     * @param throwable the exception to log, including its stack trace.
      */
```

Comment:
Same comment as above... might as well be consistent  ;-)

Reply:
Corrected.